curriculum/challenges/english/01-responsive-web-design/css-grid/use-grid-area-without-creating-an-areas-template.english.md

```diff
@@ -34,7 +34,7 @@ Using the <code>grid-area</code> property, place the element with <code>item5</c
 
 ```yml
 tests:
-  - text: <code>item5</code> class should have a <code>grid-area</code> property such that it is between the third and fourth horizontal lines and between the first and fourth vertical lines.
+  - text: <code>item5</code> class should have a <code>grid-area</code> that begins on the third horizontal line, ending on the fourth horizontal line and begins on the first vertical line, ending on the fourth vertical line.
```

Comment:
```suggestion
  - text: The <code>item5</code> class should have a <code>grid-area</code> to make it fill the whole area between the third and fourth horizontal lines, and first and fourth vertical lines.
```

This seems quite long for some test text - I tried to shorted it. What do you think of this suggestion @AnonymousRepositoryStuff? Or is that still confusing?

Reply:
I like what you have written. Much clearer than the original in my opinion.

Reply:
You can make the change if you want by clicking "Commit suggestion"